docs/dashboards/overview-dashboard.asciidoc

```diff
@@ -0,0 +1,42 @@
+[[overview-dashboard]]
+= Overview dashboard
+
+The Overview dashboard provides a high-level snapshot view of detections, external alerts, and event trends. These trends are useful to assess overall system health and find anomalies that may require further investigation.
+
+image::images/overview-pg.png[Overview dashboard]
+
+[discrete]
+== Live feed
+
+From the live feed on the Overview dashboard, you can quickly access recently created cases, favorited timelines, and the latest {elastic-sec} news.
+
+TIP: The *Security news* section provides the latest {elastic-sec} news to stay informed on new developments, learn about {elastic-sec} features, and more.
+
+image::images/live-feed-ov-page.png[Overview dashboard with live feed section highlighted]
+
+[discrete]
+== Histograms
+
+Time-based histograms show you the number of detections, alerts, and events that have occurred within the selected time range. To focus on areas of interest in time-based histograms, select a region to reflect a date range, or select a preset value in the timepicker. In the *Stack by* dropdown, you can select specific parameters to visualize individual counts. For example, in the Detection alert trend histogram, stack by `kibana.alert.rule.name` to display the total counts by alert name within the specified time frame.
+
+TIP: All Elastic Security histograms, graphs, and tables contain an *Inspect* button so you can examine the {es} queries used to retrieve data throughout the app.
```

Comment:
```suggestion
TIP: All {elastic-sec} histograms, graphs, and tables contain an *Inspect* button so you can examine the {es} queries used to retrieve data throughout the app.
```

Reply:
I want to point out that the `inspect` functionality is not always there. For eg, the Cases chart, and table do not have the inspect functionality due to implementation details (cases are saved objects vs a query, and saved objects do not have associated queries...at least I think that's the case, @stephmilovic would be able to confirm) so we do not have the capability to show the inspect tool for these. IDK if that should change the content of what you have here, I think it's still fair, but just a heads up, just in case...

Reply:
^ so maybe just "many" instead of "all"

Reply:
```suggestion
TIP: Many {elastic-sec} histograms, graphs, and tables contain an *Inspect* button so you can examine the {es} queries used to retrieve data throughout the app.
```